test/cmdlineTests/standard_stop_after_parsing_non_binary_output_requested/output.json

```diff
@@ -0,0 +1,9 @@
+{
+    "sources":
+    {
+        "C":
+        {
+            "id": 0
+        }
+    }
```

Comment:
For some reason with `stopAfter: parsing` we only disallow the outputs that require full compilation: https://github.com/ethereum/solidity/blob/53c305ea80a2f2cdc85c54f7e415dcc6c7758b7b/libsolidity/interface/StandardCompiler.cpp#L956-L960

We still allows those that require analysis (even though in that case analysis does not run) but just produce this empty output.

It's quite unexpected and easy to fix, but I decided not to do that given the reaction we got to #12153. So this test just confirms the current situation.

Reply:
I actually added a fix for `--combined-json` in https://github.com/ethereum/solidity/pull/14329/files#r1229736471 because it was very easy.